tests/test_pdf2html.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+import tools.pdf2txt
+
+
+class TestHtml(unittest.TestCase):
```

Comment:
Nice with a system test! 
Are you sure that it tests all cases of your new implementation of q(s)?